src/pages/PremiumVideoCall/components/Buttons/ToogleScreenShareButton/ToggleScreenShareButton.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * This file was modified by
+ * Mattia Primavera <sconqua@gmail.com>
+ */
+
+import React from 'react'
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles'
+
+import Button from '@material-ui/core/Button'
+import ScreenShareIcon from '../../../icons/ScreenShareIcon'
+import Tooltip from '@material-ui/core/Tooltip'
+
+import useScreenShareParticipant from '../../../hooks/useScreenShareParticipant/useScreenShareParticipant'
+import useVideoContext from '../../../hooks/useVideoContext/useVideoContext'
+
+export const SCREEN_SHARE_TEXT = 'Share Screen'
+export const STOP_SCREEN_SHARE_TEXT = 'Stop Sharing Screen'
+export const SHARE_IN_PROGRESS_TEXT =
+    'Cannot share screen when another user is sharing'
+export const SHARE_NOT_SUPPORTED_TEXT =
+    'Screen sharing is not supported with this browser'
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        button: {
+            margin: theme.spacing(1),
+            '&[disabled]': {
+                color: '#bbb',
+                '& svg *': {
+                    fill: '#bbb',
+                },
+            },
+        },
+    })
+)
+
+export default function ToggleScreenShareButton(props: { disabled?: boolean }) {
+    const classes = useStyles()
+    const screenShareParticipant = useScreenShareParticipant()
+    const { toggleScreenShare } = useVideoContext()
+    const disableScreenShareButton = Boolean(screenShareParticipant)
+    const isScreenShareSupported =
+        navigator.mediaDevices && navigator.mediaDevices.getDisplayMedia
+    const isDisabled =
+        props.disabled || disableScreenShareButton || !isScreenShareSupported
+
+    let tooltipMessage = ''
+
+    if (disableScreenShareButton) {
```

Comment:
to avoid `if` use this syntax
`disableScreenShareButton && (tooltipMessage = SHARE_IN_PROGRESS_TEXT);`